projects/plugins/jetpack/modules/sso.php

```diff
@@ -98,6 +209,138 @@ public static function get_instance() {
+		);
+		return $columns;
+	}
+
+	/**
+	 * Check if a given user is invited to the site.
+	 *
+	 * @access private
+	 * @static
+	 * @param int $user_id The user ID.
+	 *
+	 * @return {false|string} returns the user slug if the user is invited, false otherwise.
+	 */
+	private static function has_pending_wpcom_invite( $user_id ) {
+		$blog_id = Jetpack_Options::get_option( 'id' );
+
+		$user     = get_user_by( 'id', $user_id );
+		$url      = '/sites/' . $blog_id . '/invites/is-invited';
+		$url      = add_query_arg(
+			array(
+				'email_or_username' => $user->user_email,
+			),
+			$url
+		);
+		$response = Client::wpcom_json_api_request_as_user(
+			$url,
+			'v2',
+			array(),
+			null,
+			'wpcom'
+		);
+		if ( is_wp_error( $response ) ) {
+			return false;
+		}
+
+		if ( 200 !== $response['response']['code'] ) {
+			return false;
+		}
+
+		return json_decode( $response['body'], true )['slug'];
+	}
+
+	/**
+	 * Show Jetpack SSO user connection status.
+	 *
+	 * @param string $val HTML for the column.
+	 * @param string $col User list table column.
+	 * @param int    $user_id User ID.
+	 *
+	 * @return string
+	 */
+	public function jetpack_show_connection_status( $val, $col, $user_id ) {
+		if ( 'user_jetpack' === $col && Jetpack::connection()->is_user_connected( $user_id ) ) {
+			$connection_html = sprintf(
+				'<button disabled title="%1$s" class="jetpack-sso-invitation">%2$s</button>',
+				esc_attr__( 'This user is connected and can log-in to this site.', 'jetpack' ),
+				esc_html__( 'Connected', 'jetpack' )
+			);
+			return $connection_html;
+		} else {
+			$has_pending_invite = self::has_pending_wpcom_invite( $user_id );
+			if ( $has_pending_invite ) {
+				$connection_html = sprintf(
+					'<button disabled title="%1$s" class="jetpack-sso-invitation sso-pending-invite">%2$s</button>',
+					esc_attr__( 'This user didn&#8217;t accept the invitation to join this site yet.', 'jetpack' ),
+					esc_html__( 'Pending invite', 'jetpack' )
+				);
+				return $connection_html;
+			}
+			$connection_html = sprintf(
+				'<form method="post" action="%s">
+					<input type="hidden" name="user_id" value="%s" />
+					%s
+					<input type="hidden" name="action" value="jetpack_invite_user_to_wpcom" />
+					<input type="hidden" name="request" value="reinvite" />
+					<button type="submit" class="jetpack-sso-invitation sso-disconnected-user" title="%s">%s</button>
+				</form>',
+				admin_url( 'admin-post.php' ),
+				esc_attr( $user_id ),
+				wp_nonce_field( 'jetpack-sso-invite-user', '_wpnonce', true, false ),
+				esc_attr__( 'This user didn&#8217;t accept the invitation to join this site yet.', 'jetpack' ),
+				esc_html__( 'Invite', 'jetpack' )
+			);
+			return $connection_html;
+		}
+
+		return $val;
+	}
+	/**
+	 * Style the Jetpack user rows and columns.
+	 */
+	public function jetpack_user_table_styles() {
+		global $current_screen;
```

Comment:
Optional: if you wanted to use a wrapper function rather than accessing the global directly, you could use `get_current_screen()`.

Reply:
Updated this line to use `get_current_screen()`.